cpp/src/svm/svm_parameter.h

```diff
@@ -20,16 +20,25 @@ namespace SVM {
 
 /**
  * Numerical input parameters for an SVM.
+ *
+ * There are several parameters that control how long we traing. The traning
```

Comment:
```suggestion
 * There are several parameters that control how long we train. The training
```

Reply:
Apparently my spell-checker took a day off, thanks Thejaswi for spotting these!